src/amber/server/server.cr

```diff
@@ -56,28 +56,33 @@ module Amber
 
     def start
       time = Time.now
-      logger.info "#{version} serving application \"#{settings.name}\" at #{host_url}".to_s
+      log "#{version} serving application #{settings.name.capitalize.colorize(:white).mode(:underline)} at #{host_url}"
+
       handler.prepare_pipelines
       server = HTTP::Server.new(settings.host, settings.port, handler)
       server.tls = Amber::SSL.new(settings.ssl_key_file.not_nil!, settings.ssl_cert_file.not_nil!).generate_tls if ssl_enabled?
 
       Signal::INT.trap do
-        logger.info "Shutting down Amber"
+        log "Shutting down Amber"
         server.close
         exit
       end
 
-      logger.info "Server started in #{colorize(Amber.env, :yellow)}."
-      logger.info colorize("Startup Time #{Time.now - time}\n\n", :white)
+      log "Started in #{Amber.env.colorize(:yellow)}"
```

Comment:
So what happened to logger.info and logger.error etc?

Reply:
You can still use it, the `info`, and other severity can still be used. If you look at this https://github.com/amberframework/amber/pull/445/files#diff-7db448dcdac4549398cfd26fa10c9d55R20 it will show the severity when is not DEBUG

Reply:
So is `log` by default just info? It looks as if all referrences to logger.info and error have been replaced with just log?